src/Rules/ScopeRequestValidateMethods.php

```diff
@@ -0,0 +1,95 @@
+<?php declare(strict_types=1);
+
+namespace Hihaho\PhpstanRules\Rules;
+
+use Illuminate\Http\Request;
+use PhpParser\Node;
+use PhpParser\Node\Expr\MethodCall;
+use PHPStan\Analyser\Scope;
+use PHPStan\Reflection\Php\PhpFunctionFromParserNodeReflection;
+use PHPStan\Reflection\Php\PhpParameterFromParserNodeReflection;
+use PHPStan\Rules\Rule;
+use PHPStan\Rules\RuleErrorBuilder;
+use PHPStan\ShouldNotHappenException;
+
+final class ScopeRequestValidateMethods implements Rule
```

Comment:
```suggestion
/**
 * @implements \PHPStan\Rules\Rule<\PhpParser\Node\Expr\MethodCall>
 */
 final class ScopeRequestValidateMethods implements Rule
```
Think this will fix the phpstan error

Reply:
I feel like an idiot forgetting this 😓  but it only solves part of the errors